Reply/app/src/main/java/com/materialstudies/reply/ui/home/EmailViewHolder.kt

```diff
@@ -105,4 +108,12 @@ class EmailViewHolder(
         val email = binding.email ?: return
         binding.listener?.onEmailStarChanged(email, !email.isStarred)
     }
+
+    private fun updateCardViewTopLeftCornerSize(interpolation: Float) {
```

Comment:
If it's easy, could we add a comment here explaining why this is necessary. Basically - to have the container transform pick up the correct shape appearance since it doesn't understand interpolation? And maybe something along the lines of - if you don't need this work around, prefer to use a shape's `interpolation` property?

Reply:
good idea!